src/main/kotlin/domain/RandomCardPicker.kt

```diff
@@ -1,10 +1,11 @@
 package domain
 
-class RandomCardPicker : CardDrawer {
+class RandomCardPicker : CardPicker {
+    private val cardDeck =
+        CardCategory.values().flatMap { cardCategory -> CardNumber.values().map { Card.of(cardCategory, it) } }
+
     override fun draw(): Card {
-        val randomCardCategory = CardCategory.values().random()
-        val randomCardNumber = CardNumber.values().random()
-        return Card.of(cardCategory = randomCardCategory, cardNumber = randomCardNumber)
+        return cardDeck.shuffled()[FIRST]
```

Comment:
```suggestion
        return cardDeck.shuffled().first()
```
이미 첫번째 카드를 가져오는 함수가 있어요!

Reply:
수정하였습니다!